pingdom/src/main/java/com/pulse/dbcp/DataBaseInit.java

```diff
@@ -0,0 +1,66 @@
+package com.pulse.dbcp;
+
+import com.pulse.payload.URLPayload;
+import org.apache.commons.dbcp2.BasicDataSource;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Date;
+
+
+public class DataBaseInit {
+    private ArrayList<URLPayload> urlObj;
+    public DataBaseInit(ArrayList<URLPayload> urlList) {
+        this.urlObj = urlList;
+    }
+
+
+    public void dbConnect(){
+        Connection connection = null;
+        Statement statement = null;
+        ResultSet resultSet = null;
+        try {
+
+            BasicDataSource bds = DataBaseUtility.getInstance().getBds();
+            connection = bds.getConnection();
+            //looping through array of objects of type URLPayload and inserting into database
+            for(URLPayload uObj:urlObj) {
+
+
+                String sql = "insert into URLS(user_id,url,created_at,status) values(?,?,?,?)";
```

Comment:
Is this some sort of dummy query to check if DB connection is working?

Reply:
No. I'm inserting the urls into DB. User_id is dummy here. I'll update it after getting it from cookie.